### PR TITLE
PUB-2532 | Add select and button aria params

### PR DESCRIPTION
**Pull request by karelm**
## Description
Add select and button aria params

## Context & Notes

Adding semantically accurate Aria Attributes to Button and Select and updating appropriate tests with new ui version (will publish full version before deploying, using beta to test on staging for the time being)

## Checklist

-   [ ] My code follows the code style and guidelines of this project. 
-   [ ] I have added tests to cover my changes.
-   [ ] All new and existing tests passed.
-   [ ] I have tested different user stories. 
-   [ ] I have identified similar or related features and tested they are still working.
-   [ ] I have performed a self-review of my own code.
-   [ ] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
-   [ ] I have commented my code, particularly in hard-to-understand areas.
-   [ ] I kept accessibility in mind by following the [a11y checklist.](https://www.notion.so/buffer/Workflow-Checklist-e64d86eb795140bcbfdc16d1c72e573f)
-   [ ] I have considered [security, abuse & compliance](https://www.notion.so/buffer/Engineering-Wiki-f34142d290304c35bebadf76cc9cc89e#cc6dcc7617184227b77da2e1b262a563) implications of my changes.

#### Staging Deployment

To visit the URL of the staging deployment, please click "Show all checks" at the bottom of this PR and click "details" next to `bufferbotbrains/cicd-buffer-publish-legacy`

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2158,10 +2158,10 @@
     "@bufferapp/nav-sidebar" "^1.11.0"
     numeral "2.0.6"
 
-"@bufferapp/ui@5.34.2":
-  version "5.34.2"
-  resolved "https://registry.yarnpkg.com/@bufferapp/ui/-/ui-5.34.2.tgz#87b2b836bef0b4357a99328641b74b628cb3be2a"
-  integrity sha512-2sjiYhhwTHqyDFuBc+2A+FFtXmxH/ECEVEtWj6ssmiylofZNI597X/QSQwB+6VzHjqJib7M6OGnDCQFkdi3ZTw==
+"@bufferapp/ui@5.34.5-beta.c5190f1":
```
Comment (albennett) [inline yarn.lock:2161]: The split button refactor is looking great, thanks @karelm 🙌 When testing with a screen reader, there looks to be some issues with the select dropdown. I'm struggling to select items in the dropdown. This seems to be a separate issue though with the select component.

Comment (karelm) [reply yarn.lock:2161]: hey @albennett is that also present in the existing select component in publish or not?

Comment (albennett) [reply yarn.lock:2161]: Yes, there seems to be some accessibility issues with the Select component (also seeing it in the existing select component in the composer). I'll add a jira card for it.

**Comment by karelm** [comment]
Hey @letsbleachthis if you could get some eyes on the bufferbot brains branch I would appreciate it: https://task-add-select-aria-params-publish.dev.buffer.com/

**Comment by letsbleachthis** [comment]
> Hey @letsbleachthis if you could get some eyes on the bufferbot brains branch I would appreciate it: https://task-add-select-aria-params-publish.dev.buffer.com/

Thanks, @karelm! It looks good to me, and I've pinged James to give him a heads up.

**Pull request merged**